pages/13.contributing/06.Writer/docs.md

```diff
@@ -32,6 +32,126 @@ Please note that we do check to ensure that articles are not copied from other l
+  - Articles that do not have a direct connection to Mautic;
+  - Self-promotional articles (example: an article about a plugin or service written by the developer themself, or an employee).  Even if the plugin or service in question is free, the article will not be published;
+  - Articles that draw attention to issues/problems/concerns.  Mautic has other channels for that, but that isn't the purpose of the Community Blog;
+  - Articles that are too advertorial in style – rather than blatantly plug the writer’s own business, articles should employ the 'attraction' marketing principle, i.e.  the author’s expertise is evident, but not obtrusive.  For example, instead of writing about 'Why you need a marketing expert' or 'What a marketing expert can do for you', an article on 'Best practices for marketing’ would work better;
+  - Press Releases submitted as articles;
+  - Articles that are too similar to existing topics – check for similar content  [on our blog][community-blog]
+  - Articles that require too much editing.
+
+Our expectation is for articles submitted to be as 'publication ready’ as possible.  Any submissions requiring significant editing may not be considered.  This could include overly long articles or previously published/written items which require excessive modifications.  Previously published/written items are fine, provided that, prior to submission, the author has made the necessary edits in accordance with these guidelines.
+
+Articles which are not in accordance with these guidelines may be rejected!
+
+###### Interaction with readers
+
+Articles will also allow readers to add comments, and we definitely favor articles that will generate constructive discussion.  The Mautic Community Blog’s most popular articles will likely  succeed in encouraging discussions with readers, so keep this in mind when writing your article.
+
+##### Pre-launch article submission process – what we require
+
+For all articles we require you to submit your pitch via the [Editorial Calendar Jira Board][jira-board].
+
+Your pitch will be moved into either the ‘forming’ stage (where we will ask you to provide more information, research or background detail to help us determine whether it is appropriate for the Community Blog) or directly to the ‘writing’ stage.   
+
+When your pitch is moved into the writing stage, you take up the responsibility for writing the content within the time frame on the card.   Please note the draft due date – this is when you need to have finished writing the article.  
+
+##### Style Guide summary
+
+  - All spelling should be in US English.
+  - Write numbers one to nine as words and numbers 10 and above as numerals.
+  - Where possible, avoid mentioning specific companies or organizations in your article unless it is relevant to your subject.
+  - If you want to emphasize a phrase or word, use italics rather than underlining.  Readers assume underlined text is hyperlinked.
```

Comment:
```suggestion
  - If you want to emphasize a phrase or word, use italics rather than underlining. Readers assume underlined text is hyperlinked.
```